### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 23

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#295] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320634-c9369baf-ce76-40b3-8f20-fde4dd0bc63e.png)

**Comment by LuginiAndrea** [comment]
$R$ non è un ordine parziale in quanto viola la transitività: $3R4, \nexists 3R5$
$R \circ R = \left(1,4\right), \left(2, 5\right), \left(3,5\right)$
$R^T = R \cup \left(1,4\right), \left(1,5\right), \left(2,5\right), \left(3,5\right)$
